src/offset/local.rs

```diff
@@ -193,6 +207,46 @@ impl TimeZone for Local {
     }
 }
 
+#[cfg(all(feature = "test-override"))]
+thread_local!(static OVERRIDE: std::cell::RefCell<Option<DateTime<Local>>> = std::cell::RefCell::new(None));
```

Comment:
nitpick: "Override" is a broad term, can this named `LOCAL_NOW_OVERRIDE` ? 

Also, could this somewhat complex global variable include a `///` code comment describing why it's necessary?